### PR TITLE
docs(cn): translate src/content/glossary.md

### PR DESCRIPTION
**Pull request by daxiaoxiaodejia**
_describe your changes..._

- [ ] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [ ] Make sure your PR complies with the [writer's guide][2].
- [ ] Review the diff carefully as sometimes this can reveal issues.
- [ ] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

File: src/content/glossary.md [hunk 1]
```diff
@@ -11,88 +11,88 @@ contributors:
...
+- [__Loaders__](/concepts/loaders): loader 用于对模块的源代码进行转换。loader 可以使你在 `require()` 或"加载"模块时预处理文件。类似于一个 “task-runner”。
+- [__Lazy Loading__](/guides/lazy-loading): 对应用程序的部分（块）进行懒加载的过程。换句话说，只有我们在真正需要它们的时候才进行加载。
 
 
 ## M
 
-- [__Module__](/concepts/modules): Discrete chunks of functionality that provide a smaller surface area than a full program. Well-written modules provide solid abstractions and encapsulation boundaries which make up a coherent design and clear purpose.
-- [__Module Resolution__](/concepts/module-resolution/): A module can be required as a dependency from another module and a resolver is a library which helps in locating a module by its absolute path. Modules are searched for inside all directories specified in `resolve.modules`.
-- [__Manifest__](/concepts/manifest): The runtime will use it to resolve and load modules once they've been bundled and shipped to the browser.
+- [__Module__](/concepts/modules): 离散功能块相比于完整程序提供了更小的接触面。精心编写的模块提供了可靠的抽象和封装界限，使得应用程序中每个模块都具有条理清楚的设计和明确的目的。
```
Comment (lcxfs1991) [inline src/content/glossary.md:58]: 这里的意思是说，Module 是descrete chunks

File: src/content/glossary.md [hunk 2]
```diff
@@ -11,88 +11,88 @@ contributors:
...
-- [__Output__](/concepts/output): Option(s) specifying where to output the compiled files to disk.
-  > _Note, that while there can be multiple entry points, only one output configuration is specified._
+- [__Output__](/concepts/output): 配置项指定将编译的文件输出到磁盘的位置。
+  > _注意,即使可以存在多个入口起点，但只指定一个输出配置。_
 
 
 ## P
 
-- [__Plugin__](/concepts/plugins): A JavaScript object that has an `apply` property. This `apply` property is called by the webpack compiler, giving access to the entire compilation lifecycle. These packages will typically extend compilation functionality in one way or another.
+- [__Plugin__](/concepts/plugins): webpack 插件是一个具有 `apply` 属性的 JavaScript 对象。 `apply` 属性会被 webpack compiler 调用，并且 compiler 对象可在整个编译生命周期访问。这些包通常会以某种方式扩展编译功能。
```
Comment (lcxfs1991) [inline src/content/glossary.md:71]: 是插件可访问生命周期，不是compiler

File: src/content/glossary.md [hunk 3]
```diff
@@ -11,88 +11,88 @@ contributors:
...
 ## V
 
-- [__Vendor Entry Point__](/concepts/entry-points/#separate-app-and-vendor-entries): Create dependency graphs starting at both `app.js` and `vendors.js`. These graphs are completely separate and independent of each other to allow leverage of `CommonsChunkPlugin` and extract any vendor references from your app bundle into your vendor bundle. Helps achieve a common pattern in webpack known as [long-term vendor-caching](/guides/caching/).
+- [__Vendor Entry Point__](/concepts/entry-points/#separate-app-and-vendor-entries): 从 `app.js` 和 `vendors.js` 开始创建依赖图。这些依赖图完全是分开且独立的，允许使用 `CommonsChunkPlugin`，并将应用程序包的任何供应商参考提取到你的供应商包中。有助于在 webpack 中实现一种称为 [长期供应商缓存](/guides/caching/) 的常见模式。
 
 
 ## W
 
-- [__webpack__](/): A highly configurable [module](/concepts/modules) bundler for modern JavaScript applications.
+- [__webpack__](/): 一个用于现代 JavaScript 应用程序的高度可配置的 [module]（/concepts/modules）捆绑包。
```
Comment (lcxfs1991) [inline src/content/glossary.md:98]: 捆绑包 -> 打包工具

File: src/content/glossary.md [hunk 4]
```diff
@@ -11,88 +11,88 @@ contributors:
...
+- [__Target__](/configuration/target/): 用户配置的部署目标 [此处列出](/configuration/target/) 用于为特定环境编译，如浏览器、 NodeJS 或 Electron。
+- [__Tree Shaking__](/guides/tree-shaking/): 删除未使用 / 多余的代码，或者更准确地说，实时代码导入。像 webpack 这样的编译器将通过分析各种 `import` 语句和导入代码的使用情况，来确定实际使用了依赖项的哪些部分来实现这一点，删除那些没有使用的 “树” 的部分。
 
 
 ## V
 
-- [__Vendor Entry Point__](/concepts/entry-points/#separate-app-and-vendor-entries): Create dependency graphs starting at both `app.js` and `vendors.js`. These graphs are completely separate and independent of each other to allow leverage of `CommonsChunkPlugin` and extract any vendor references from your app bundle into your vendor bundle. Helps achieve a common pattern in webpack known as [long-term vendor-caching](/guides/caching/).
+- [__Vendor Entry Point__](/concepts/entry-points/#separate-app-and-vendor-entries): 从 `app.js` 和 `vendors.js` 开始创建依赖图。这些依赖图完全是分开且独立的，允许使用 `CommonsChunkPlugin`，并将应用程序包的任何供应商参考提取到你的供应商包中。有助于在 webpack 中实现一种称为 [长期供应商缓存](/guides/caching/) 的常见模式。
```
Comment (lcxfs1991) [inline src/content/glossary.md:93]: 供应商参考 -> 供应商(vendor)引用。

File: src/content/glossary.md [hunk 5]
```diff
@@ -11,88 +11,88 @@ contributors:
...
 ---
 
-This index lists common terms used throughout the webpack ecosystem.
+该索引列出了整个 webpack 生态系统中的常用术语。
 
 
 ## A
 
-- [__Asset__](/guides/asset-management/): This a general term for the images, fonts, media, and any other kind of files that are typically used in websites and other applications. These typically end up as individual files within the [output](/glossary/#o) but can also be inlined via things like the [style-loader](/loaders/style-loader) or [url-loader](/loaders/url-loader).
+- [__Asset__](/guides/asset-management/): 资源是对图像、字体、媒体和任何其他类型文件的统称，通常用于网站和其他应用程序中。这些文件通常在 [output](/glossary/#o) 中最终输出为单独的文件，但也可以通过诸如 [style-loader](/loaders/style-loader) 或 [url-loader](/loaders/url-loader) 之类的方法内联。
```
[the remark this answers is not in the document]
Comment (daxiaoxiaodejia) [reply src/content/glossary.md:19]: markdown 里面都只占用了一行。中文比英文要少，这种情况我可以从中间断下来吗？还是翻译规则上说的用空行？这里我不太明白空行是什么意思？

**Pull request merged**